src/Symfony/Component/Mailer/Bridge/Mailgun/Mailgun.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Mailgun;
+
+/**
+ * @author Michael Garifullin <garifullin@gmail.com>
+ *
+ * @experimental in 4.3
+ */
+class Mailgun
+{
+    public const REGION_EU = 'EU';
+    public const REGION_US = 'US';
+
+    private const SMTP_DOMAIN_US = 'smtp.mailgun.org';
+    private const SMTP_DOMAIN_EU = 'smtp.eu.mailgun.org';
+
+    private const ENDPOINT_DOMAIN_EU = 'api.eu.mailgun.net';
+    private const ENDPOINT_DOMAIN_US = 'api.mailgun.net';
```

Comment:
If we used arrays for the mapping here, we could save some `if`/`else` blocks below.

```php
private const SMTP_HOSTS = [
    self::REGION_EU => 'smtp.eu.mailgun.org',
    self::REGION_US => 'smtp.mailgun.org',
];

private const ENDPOINT_DOMAINS = [
    self::REGION_EU => 'api.eu.mailgun.net',
    self::REGION_US => 'api.mailgun.net',
];
```